src/pages/signup-affiliates/index.tsx

```diff
@@ -139,6 +142,40 @@ const AffiliateSignup = () => {
     const [email_error_msg, setEmailErrorMsg] = useState('')
     const [submit_status, setSubmitStatus] = useState('')
     const [submit_error_msg, setSubmitErrorMsg] = useState('')
+    const { send } = useDerivWS()
+
+    const getVerifyEmailRequest = (formatted_email) => {
+        const cookies = getCookiesFields()
+        const cookies_objects = getCookiesObject(cookies)
+        const cookies_value = getDataObjFromCookies(cookies_objects, cookies)
+        const affiliate_url = window.location.pathname.match('/signup-affiliates/')
+        let account_status = 'account_opening'
+        const token = queryParams.get('t')
+
+        if (token && cookies_value.utm_campaign === 'CellXpert') {
+            cookies_value.utm_medium = 'affiliate'
+        }
+        if (affiliate_url) {
+            account_status = 'partner_account_opening'
+            delete cookies_value.utm_campaign
+            delete cookies_value.utm_medium
+            cookies_value.utm_source = 'null'
+        }
+
+        if (!token) {
+            delete cookies_value.utm_campaign
+            delete cookies_value.utm_medium
+            cookies_value.utm_source = 'null' //passing null as the cookies takes the affiliates token value when signed up without affiliate token
+        }
```

Comment:
We don't need these extra rules. Since now we are already in the affiliates page, we can just pass the data directly. 

`account_status` will always be `partner_account_opening` because we are in the `signup-affiliates` page